packages/core/src/shared/constants.ts

```diff
@@ -129,6 +129,7 @@ export const ecsIamPermissionsUrl = vscode.Uri.parse(
  */
 export const CLOUDWATCH_LOGS_SCHEME = 'aws-cwl' // eslint-disable-line @typescript-eslint/naming-convention
 export const AWS_SCHEME = 'aws' // eslint-disable-line @typescript-eslint/naming-convention
+export const AMAZON_Q_DIFF_SCHEME = 'amazon-q-view-diff' // eslint-disable-line @typescript-eslint/naming-convention
```

Comment:
eslint-disable-line was a compromise to avoid code churn, not intended to be continued for new symobls

is "view" an imporant thing to mention in the scheme name?

```suggestion
export const amazonQDiffScheme = 'amazon-q-diff'
```